web-registry/src/types/ledger/ecocredit.ts

```diff
@@ -72,3 +72,70 @@ export interface QueryBasketsResponse {
   baskets: Basket[];
   pagination?: PageResponse;
 }
+
+export interface IBasket {
```

Comment:
this is just a temporary thing until we update the regen-js api package. I leave it as it is, it will be removed next week.

without intention to introduce new nomenclature. We'll talk about that in due course, about naming conventions, etc.

initially I had that interface defined with the name Basket, with camelCase, but then another one came in with snake_case, and I simply renamed mine

Reply:
by the way, they finally changed :)